easybuild/easyconfigs/m/Minimac/Minimac-2013.7.17-goolf-1.4.10.eb

```diff
@@ -0,0 +1,45 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock='MakeCp'
+
+name = 'Minimac'
+version = '2013.7.17'
+versionsuffix = '-beta'
```

Comment:
makes sense, yeah

We should be using the datestamp in the source tarball name too, but that breaks auto-downloading.